src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -941,4 +944,10 @@
   <data name="CloseSnippetsPaneButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
     <value>Close pane</value>
   </data>
-</root>
+  <data name="TabMoveLeft" xml:space="preserve">
+    <value>Move left</value>

```

Comment:
I'd prefer "Move tab left" and "Move tab right" personally, but in the absence of other opinions i can stand down